### PR TITLE
81 ffmc UI

### LINKED ISSUE #81: FFMC UI

### PR DESCRIPTION
**Pull request by Davis245**
Created the ffmc visualization using html, css, and javaScript:
      - The html is on the fire.html in the ffmc widget spot
      - The css is in static/css/ffmc.css
      - The javaScript is in static/JavaScript/ffmc.js
      
The functionality can you tested by manipulating the example value in the ffmc.js file.

### CODE REVIEW

**Comment by robertyac** [comment]
I can't get onto the page since home doesn't exist now. I tried all your other pull requests too and @cmcahoon01 's PR and I get the same thing. Looks like the settings or urls files need to be updated.

Screenshot 2024-03-06 at 7 55 13 PM

**Comment by cmcahoon01** [comment]
I've just been struggling through the same issue. All references to "home" in views.py need to be replaced with weather or fire

File: Site/bc_weather_station_dashboard/website/views.py [hunk 1]
```diff
@@ -8,10 +8,12 @@
 from .models import WeatherStation
 
 
-# Create your views here.
 def home(request):
     return render(request, "home.html", {})
```
Comment (carsondrobe) [inline Site/bc_weather_station_dashboard/website/views.py]: Change this to "weather.html"

**Review by carsondrobe — CHANGES_REQUESTED** [review]

File: Site/bc_weather_station_dashboard/website/views.py (hunk 1 above)
Comment (carsondrobe) [reply Site/bc_weather_station_dashboard/website/views.py]: return render(request, "weather.html", {})

**Review by carsondrobe — APPROVED** [review]
WORKS

**Review by robertyac — APPROVED** [review]
lgtm

**Pull request merged**